client-device-connection-infinispan/pom.xml

```diff
@@ -52,18 +50,8 @@
       <groupId>org.infinispan</groupId>
       <artifactId>infinispan-core</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.eclipse.hono</groupId>
-      <artifactId>hono-client-common</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>io.vertx</groupId>
-      <artifactId>vertx-web</artifactId>
-    </dependency>
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-    </dependency>
+
+    <!-- TODO: PR review, should this not come transiently from the client-device-connection module? -->
```

Comment:
we explicitly define all components  that we depend on. We feel that this more robust than relying on transitive dependencies providing the needed classes. This way we also have more control over versions of these deps and conflicts show up more reliably.